### DISCUSSION TITLE
Parser for HTML

### DISCUSSION THREAD

**Discussion by cagnulein** [cagnulein/qdomyos-zwift#355] [Ideas]
@lifof which html parser should we use in qt?

**Comment by lifof** [comment]
@cagnulein 
I never used an HTML parser before, but we'll figure something out.
Is this for a new functionality ? where is it needed ?

**Reply by cagnulein** [reply]
Yes for #354 , @p3g4asus suggests to use regex

**Reply by lifof** [reply]
OK so from what I understood there's a need to get the HTML table from [this website](https://app.homefitnessbuddy.com/peloton/powerzone/) ?

**Comment by cagnulein** [comment]
Yep

Il lun 7 giu 2021, 21:39 lifof ***@***.***> ha scritto:

**Comment by cagnulein** [comment]
Basically i have to write a similar module of powerzonepack.cpp for this
site. It should be quite simple if the qt should have a html parser

Il lun 7 giu 2021, 21:44 cagnulein ***@***.***> ha scritto:

**Reply by lifof** [reply]
OK got it I'll see what I can do

**Reply by cagnulein** [reply]
@lifof , @p3g4asus told me that would try something about regex Tomorrow or in the next days. Let's wait for him,. I was just asking about advices :)

**Reply by lifof** [reply]
OK got it
